ex3.html

```diff
@@ -1,555 +1,558 @@
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Tooling
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>						
+										</select>
+									</li>
+									<li>Libraries
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>		
+										</select>
+									</li>
+									<li>Promises
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+									<li>Browser Engines
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+									<li>Frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>			
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+
+
+							<fieldset>
+								<legend>NodeJS</legend>
+								<ul>
+									<li>Backend frameworks
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>	
+										</select>
+									</li>
+									<li>Templating
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>DOM Manipulation
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>
+										</select>
+									</li>
+									<li>Unit Testing
+										<select>
+											<option selected disabled> Choose </option>
+											<option value="volvo">V1</option>
+											<option value="volvo">V2</option>
+											<option value="volvo">V3</option>				
+										</select>
+									</li>
+								</ul>
+
+							</fieldset>
+
+							<h3><a href="ex2.html" target="_top" style="float: right;">SUBMIT</a></h3>
+						</form>
+
+
+						<style type="text/css">
```

Comment:
Any particular reason for this to be defined in the HTML? Furthermore, any particular reason for it to be defined at the bottom of the HTML? Please keep in mind that in order for the page to be properly displayed, the CSS Object Model (CSSOM - the DOM's equivalent for styling) needs to be fully built (otherwise, subsequent styles may invalidate the look & feel of the page or parts of it, thus implicitly causing a re-render). Hence, stylesheets should load as fast as possible for increased performance of the initial page render. You can read more on that [here](https://developers.google.com/web/fundamentals/performance/critical-rendering-path/render-blocking-css).